app.js

```diff
@@ -129,308 +121,16 @@ let cubes = require('./routes/cube_routes');
 let users = require('./routes/users_routes');
 let devs = require('./routes/dev_routes');
 let tools = require('./routes/tools_routes');
+app.use('', require('./routes/root'));
```

Comment:
Shouldn't this come after the other app.use? Wouldn't all routes just get routed into here?